workflow-bot-app/src/main/java/com/symphony/bdk/workflow/configuration/WorkflowDeployer.java

```diff
@@ -60,119 +51,41 @@ public void addAllWorkflowsFromFolder(Path path) {
     }
   }
 
-  public void handleFileEvent(Path changedFile, WatchEvent<Path> event) throws IOException, ProcessingException {
-    if (isYaml(changedFile)) {
-      if (event.kind().equals(StandardWatchEventKinds.ENTRY_CREATE)) {
-        this.addWorkflow(changedFile, WorkflowMgtAction.DEPLOY);
-
-      } else if (event.kind().equals(StandardWatchEventKinds.ENTRY_MODIFY)) {
-        this.addWorkflow(changedFile, WorkflowMgtAction.UPDATE);
-
-      } else if (event.kind().equals(StandardWatchEventKinds.ENTRY_DELETE)) {
-
-        /*
-          The workflow id is required for the version undeployment and deletion. Querying the database by the SWADL
-          file path is expensive as the column is not indexed (and should not be indexed).
-          But this is the only way to have the workflow id as it cannot be provided in parameters because the actual
-          file is already deleted from the filesystem.
-         */
-        Optional<VersionedWorkflow> versionedWorkflow = this.versioningService.findByPath(changedFile);
-        if (versionedWorkflow.isPresent()) {
-          String workflowId = versionedWorkflow.get().getWorkflowId();
-          this.workflowEngine.undeploy(workflowId);
-          this.versioningService.delete(workflowId);
-        }
-
-      } else {
-        log.debug("Unknown event: {}", event);
-      }
-    }
-  }
-
-  public void addWorkflow(Path workflowFile, WorkflowMgtAction action) throws IOException, ProcessingException {
+  void addWorkflow(Path workflowFile) throws IOException, ProcessingException {
```

Comment:
can be private

Reply:
it is used in a test, so leave it in package level